code/api/src/migrations/20210325160707-6-crateProducts.js

```diff
@@ -0,0 +1,31 @@
+"use strict";
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable("crateProducts", {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      crateId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "crates",
+          key: "id"
+        },
+        allowNull: false
+      },
+      productId: {
```

Comment:
I would like to add details about different variable names & their role in the response/request process. (Aka, add it to eh API documentation & make it explicit)

Reply:
Could add to our data map, too.

Reply:
Yes! I think the more explicit we are upfront the better. Nice suggestion.

Reply:
Made a note to add it into the contract